css/style.css

```diff
@@ -1,37 +1,37 @@
-* {
+*{
```

Comment:
I would keep the spacing here between the class & the first brackets. IMO, it's make the CSS declaration more readable.